.changeset/strange-paws-smell.md

```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/core": minor
+"@twilio-paste/form-pill-group": minor
+---
+
+[Form Pill Group] Adding size property to the FormPillGroup component. Sizes include default and "L"
```

Comment:
suggestion(non-blocking): now variant changes the changeset should be "large". These comments are included in our release notes but very minor.

Reply:
Updated.